README.md

```diff
@@ -55,7 +54,6 @@ ShopifyAPI::Context.setup(
   scope: "read_orders,read_products,etc",
   session_storage: ShopifyAPI::Auth::FileSessionStorage.new, # See more details below
   is_embedded: true, # Set to true if you are building an embedded app
-  is_private: false, # Set to true if you are building a private app
```

Comment:
Defaults to false: https://github.com/Shopify/shopify-api-ruby/blob/9ef0f174fa15bea19c231a1ed5ef57c889edd13e/lib/shopify_api/context.rb#LL13C25-L13C25